test/e2e/view-caseload.js

```diff
@@ -29,7 +29,7 @@ describe('View your caseload flow', () => {
       await browser.url(teamDefaultUrl + '/caseload')
       pageSubtitle = await $('.govuk-caption-xl')
       pageSubtitle = await pageSubtitle.getText()
-      expect(pageSubtitle).to.equal('Team')
+      expect(pageSubtitle).to.equal('Offender Manager')
```

Comment:
Clearly the H1 was previously used to confirm the navigation went to the right page.

This causes us a problem now the H1 doesn't have that information. Looking at the HTML there is no obvious other way of finding out which organisational level each page is for - do we add something to every page to allow these tests to function?

If not, should we bother with these checks at all?